core/src/main/java/org/kabeja/entities/Hatch.java

```diff
@@ -382,4 +382,13 @@ public void setPatternScale(double patternScale) {
 	public void transform(TransformContext context) {
 
 	}
+
+	/**
+	 * @ToDo: implement this method
+	 *
+	 * This is a planar entity, therefore this method should be implemented,
+	 * otherwise OCS-coordinates will be used regardless of calling this method.
+	 */
+	@Override
+	public void toWcs() { }
```

Comment:
Can toWcs throw a `NotImplementedException` on classes where it would be needed but isn't implemented? And on all other classes it would be good if we had a short comment saying that it's not needed